src/lib/components/dashboard/credentials/CredentialList.svelte

```diff
@@ -54,6 +55,11 @@
       );
     }
   }
+
```

Comment:
can you change the function name to be recognizable also use const functionName = ()